### PR TITLE
pam-dm-update Update BXMS and BPM Suite references

### PR DESCRIPTION
**Pull request by emmurphy1**

### CODE REVIEW

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc [hunk 1]
```diff
@@ -144,7 +144,7 @@
 
 [[realtime-decision-server]]
 ==== image:images/yes.png[yes] Realtime Decision Server (noun)
-*Description*: In Red{nbsp}Hat JBoss BRMS and Red{nbsp}Hat JBoss BPM Suite, the _Realtime Decision Server_ is a standalone, built-in component that can be used to instantiate and execute rules through interfaces available for REST, JMS, or a Java client-side applications. Created as a web deployable WAR file, this server can be deployed on any web container. The current version of the Realtime Decision Server is included with default extensions for both Red{nbsp}Hat JBoss BRMS and Red{nbsp}Hat JBoss BPM Suite.
+*Description*: In Red{nbsp}Hat Decision Manager and Red{nbsp}Hat Process Automatio M, the _Realtime Decision Server_ is a standalone, built-in component that can be used to instantiate and execute rules through interfaces available for REST, JMS, or a Java client-side applications. Created as a web deployable WAR file, this server can be deployed on any web container. The current version of the Realtime Decision Server is included with default extensions for both Red{nbsp}Hat Decision Manager and Red{nbsp}Hat Process Automatio M.
```
Comment (dfitzmau) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:147]:
```suggestion
*Description*: In Red{nbsp}Hat Decision Manager and Red{nbsp}Hat Process Automation, the _Realtime Decision Server_ is a standalone, built-in component that can be used to instantiate and execute rules through interfaces available for REST, JMS, or a Java client-side applications. Created as a web deployable WAR file, this server can be deployed on any web container. The current version of the Realtime Decision Server is included with default extensions for both Red{nbsp}Hat Decision Manager and Red{nbsp}Hat Process Automation.
```

**Comment by dfitzmau** [comment]
Hi @emmurphy1 . I see 5 commits in teh hsitory, would you be OK to squash them down to 1 commit?

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc (hunk 1 above)
Comment (emmurphy1) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc]: FIxed in source and pushed.

**Review by dfitzmau — DISMISSED** [review]

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc [hunk 2]
```diff
@@ -256,6 +256,18 @@ The Red{nbsp}Hat Container Catalog no longer exists; it has become part of the R
 
 *See also*: xref:data-grid[Data Grid], xref:red-hat-jboss-data-grid[Red{nbsp}Hat JBoss Data Grid]
 
+[discrete]
```
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:259]: We don't use discrete tags anymore, can you remove this?

```suggestion

```

**Review by bergerhoffer — COMMENTED** [review]
Just one line to remove, otherwise it LGTM!

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc (hunk 2 above)
Comment (bredamc) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:259]: @emmurphy1 I think the only remaining change needed is to remove the tag in line 259. Can you please make this change and we'll try to get this PR approved? Many thanks :)

Comment (emmurphy1) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc]: @bredamc done!

**Review by dfitzmau — APPROVED** [review]

**Review by bergerhoffer — APPROVED** [review]
LGTM!

**Comment by bergerhoffer** [comment]
3 approvals merging!

Thanks again @emmurphy1 for working on this!

**Pull request merged**